test/integration/user-management-api/services/proxies_test.rb

```diff
@@ -63,9 +63,7 @@ def test_update_for_service_mesh
     params = provider_key_params.merge(proxy: { credentials_location: 'headers' })
     @service.update_column :deployment_option, 'service_mesh_istio'
 
-    assert_difference @service.proxy.proxy_configs.production.method(:count), +1 do
-      put(admin_api_service_proxy_path(params))
-    end
+    put(admin_api_service_proxy_path(params))
```

Comment:
Could you explain this change? Seems like the assertion is gone, but I can't understand what was original intention and why it can't work anymore.